Update auto-generated documentation

snoopycrimecop opened the pull request:
Repository: openmicroscopy/ome-documentation
Already up-to-date.



Generated by build [OMERO-DEV-latest-docs-autogen#1109](https://ci.openmicroscopy.org/job/OMERO-DEV-latest-docs-autogen/1109/).

----
--no-rebase

hflynn commented:
Looks good so far, can leave this open until after https://github.com/openmicroscopy/openmicroscopy/pull/5578 is merged to get the history in too

hflynn commented:
@joshmoore this adds the autogen content warning banner to the top of those other files q.v. your comment on #1801

hflynn commented on omero/users/history.rst:
```diff
@@ -1,7 +1,3 @@
-.. Content for this page should be opened as a PR against
```
I forgot to add this banner to the page in the code repo, I'll update my version history PR now to do that.

snoopycrimecop committed:
Update auto-generated documentation

Repository: openmicroscopy/ome-documentation
Already up-to-date.

Generated by OMERO-DEV-latest-docs-autogen#1109 (https://ci.openmicroscopy.org/job/OMERO-DEV-latest-docs-autogen/1109/)
